engine/baml-runtime/src/cli/serve/mod.rs

```diff
@@ -313,21 +317,7 @@ impl Server {
             tcp_listener,
             app.layer(axum::middleware::from_fn(Server::auth_middleware)),
         );
-        // TODO: we do not handle this ourselves, because tokio's default
-        // handling is pretty good on unix.
-        //
-        // Not totally sure if this WAI on
-        // windows, but there are some potential pitfalls that we can run into
-        // if we try to handle this ourselves, see
-        // https://docs.rs/tokio/latest/tokio/signal/fn.ctrl_c.html#caveats
-        //
-        // Namely- we need to ensure resilient delivery of Ctrl-C to everything, and I
-        // suspect we need to do a bit of work to ensure that we handle that bookkeeping
-        // correctly. Shutting down the BAML runtime, tokio runtime, _and_ axum is not
-        // super straightforward, because I don't know how much is handled for us
-        // out of the box.
-        //
-        // .with_graceful_shutdown(signal::ctrl_c());
```

Comment:
Can we keep these comments around in this file? Even if we push the libc signal handler stuff to language_client_python/src/lib.rs. Mostly just because this is because a good reminder for myself about how careful we should be about touching signal handling code here.